### PR TITLE
My account support

### PR DESCRIPTION
**Pull request by kevinreiss**
Makes a start on #330. Currently this has account information displayed for an authenticated CAS user at /account.

### CODE REVIEW

**Comment by kevinreiss** [comment]
If you think this is a decent start let me rebase this before it gets merged.

**Comment by kevinreiss** [comment]
It is strange the single failing test passes on my local environment.

**Comment by kevinreiss** [comment]
For some reason my attempt to mock API calls to the account service and allow other external API calls didn't run as I expected in CI.

File: Gemfile [hunk 1]
```diff
@@ -70,11 +70,13 @@ gem 'coveralls', require: false
...
 group :development do
   gem 'capistrano-rails', '~> 1.1.1'
-  gem 'pry-byebug'
 end
 
-group :test, :development do
+group :test do
 #	gem "jettywrapper", "~> 1.7"
 #	gem 'rspec-rails', '~> 3.0.0'
+  gem 'pry-byebug'
```
Comment (tampakis) [inline Gemfile]: I use pry-byebug in development mode too.

Comment (kevinreiss) [inline Gemfile]: Okay. I ran into some weirdness with getting the test suite to pass on Travis based on how those groups were constructed. What gems should we be sharing between the two envs?

Comment (tampakis) [inline Gemfile]: That's the only one. I actually only use it in development.

**Comment by kevinreiss** [comment]
Thanks to @tpendragon @escowles for the work from plum/pom to mock cas sessions. I used that strategy to mock cas logins in this PR.

File: Gemfile (hunk 1 above)
Comment (kevinreiss) [inline Gemfile]: Woops I forgot to change this. I'll put in in a group for test and development.

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -51,10 +52,31 @@ def locate_link_with_gylph location, bib, library=nil
...
+  #       cn_browse_link = link_to('[Browse]', "/browse/call_numbers?q=#{holding['call_number']}", class: 'browse-cn',
+  #                           'data-toggle' => "tooltip", 'data-original-title' => "Browse: #{holding['call_number']}",
+  #                           title: "Browse: #{holding['call_number']}")
+  #       cn = "Call Number: #{holding['call_number']} #{cn_browse_link}"
+  #       info << content_tag(:li, cn.html_safe, class: 'holding-call-number')
+  #     end
+  #     block << content_tag(:li, content_tag(:ul, info.html_safe, holding_id: id), class: 'holding-block') unless info.empty?
+  #   end
+  # end
+
```
Comment (tampakis) [inline app/helpers/application_helper.rb]: leftover cruft

**Pull request merged**